### ISSUE TITLE
"Applied Accessibility - Wrap Content in the article Element" cannot be completed because it always thinks there are <div>s

### ISSUE THREAD

**Issue by SuperGeniusZeb** [freeCodeCamp/learn#127]
https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/wrap-content-in-the-article-element/

This lesson cannot be completed because the second test ("Your code should not have any `div` tags.") always fails, no matter what the content of the editor is.

Tested in Firefox Nightly.

Related:
- #128
- #129
- #130
- #134
- #136

**Comment by Bouncey** [comment]
I cannot reproduce, can you share your code please?

**Comment by SuperGeniusZeb** [comment]
```
<h1>Deep Thoughts with Master Camper Cat</h1>
<main>
  <article>
    <h2>The Garfield Files: Lasagna as Training Fuel?</h2>
    <p>The internet is littered with varying opinions on nutritional paradigms, from catnip paleo to hairball cleanses. But let's turn our attention to an often overlooked fitness fuel, and examine the protein-carb-NOM trifecta that is lasagna...</p>
  </article>

  <img src="samuraiSwords.jpeg" alt="">

  <article>
    <h2>Defeating your Foe: the Red Dot is Ours!</h2>
    <p>Felines the world over have been waging war on the most persistent of foes. This red nemesis combines both cunning stealth and lightening speed. But chin up, fellow fighters, our time for victory may soon be near...</p>
  </article>

  <img src="samuraiSwords.jpeg" alt="">

  <article>
    <h2>Is Chuck Norris a Cat Person?</h2>
    <p>Chuck Norris is widely regarded as the premier martial artist on the planet, and it's a complete coincidence anyone who disagrees with this fact mysteriously disappears soon after. But the real question is, is he a cat person?...</p>
  </article>
</main>
```
The `div` test also fails if there is nothing in the editor.

I just tested in Chromium, and the issue is not happening there. Can you reproduce the issue in Firefox?

Also, I have noticed that the website switched back and forth between 2 designs yesterday. Does the website look like this to you right now?
![image](https://user-images.githubusercontent.com/19592990/40842621-153344d2-6574-11e8-95a4-9871f4040928.png)

(The other design has a second row under the header with the <kbd><samp>Run the Tests</samp></kbd> button in it, and the tests show up under the preview of the project instead of the description.)

**Comment by SuperGeniusZeb** [comment]
I should have clarified, but I was using Firefox Nightly. I recently switched to Firefox Beta, and the issue is still occurring there. I am using Antergos Linux.

I tried disabling various Firefox add-ons, and it turns out that Plasma Integration is the add-on causing the issue:

https://addons.mozilla.org/en-US/firefox/addon/plasma-integration/

Disabling the add-on makes the `<div>` tests immediately start working without even refreshing the lesson page. Likewise, enabling the add-on immediately causes the `<div>` tests to stop working.

I have no idea why this add-on causes an issue. This issue only seems to happen with tests for `<div>`s, not any other HTML element.

The corresponding Chromium extension does not cause any issues with the freeCodeCamp tests. I have no idea if it is because of a difference between Firefox and Chromium, a difference between the Firefox add-on and its matching Chromium extension, both, or neither.

**Comment by raisedadead** [comment]
See https://github.com/freeCodeCamp/learn/issues/136#issuecomment-398936869
